src/framework/_typography.scss

```diff
@@ -134,24 +152,19 @@ $type-sizes: (
 
     .type-#{$key} {
       @include fz($key);
+
+      // Media queries
+      @include for-each-breakpoints using ($breakpoint) {
+        @include fz($key, 'em', $breakpoint);
+      }
     }
 
     .type-rem-#{$key} {
       @include fz($key, 'rem');
-    }
-  }
-
-  // Media queries
-  @include for-each-breakpoints using ($breakpoint) {
-    @each $type-size in $type-sizes {
-      $size: nth($type-size, 1);
-
-      .type-#{$size}--#{$breakpoint} {
```

Comment:
Maybe we should keep this declaration to force a type size on some element.
```html
<span class="type-headline-1 type-headline-3--m">Example</span>
```

Reply:
Wouldn't it become a little bit too complicated to have both media queries for the class and by modifiers?

Reply:
Actually, it is a good idea so this feature is not a breaking change. I'll update the PR.

Reply:
After some discussion, we are dropping this behavior for good, it will be easier on the long run.